src/js/utils/browser.ts

```diff
@@ -83,32 +81,7 @@ export function isIframe(): boolean {
     }
 }
 
-export function flashVersion(): number {
-    if (isAndroid()) {
-        return 0;
-    }
-
-    const plugins = navigator.plugins;
-    let flash;
-
-    if (plugins) {
-        flash = plugins.namedItem('Shockwave Flash');
-        if (flash && flash.description) {
-            return parseFloat(flash.description.replace(/\D+(\d+\.?\d*).*/, '$1'));
-        }
-    }
-
-    if (typeof window.ActiveXObject !== 'undefined') {
-        try {
-            flash = new window.ActiveXObject('ShockwaveFlash.ShockwaveFlash');
-            if (flash) {
-                return parseFloat(flash.GetVariable('$version').split(' ')[1].replace(/\s*,\s*/, '.'));
-            }
-        } catch (e) {
-            return 0;
-        }
-
-        return flash;
-    }
+// Always returns 0 as flash support is discontinued
+export function flashVersion(): 0 {
```

Comment:
Do we need to keep this function?

Reply:
Yep. I sync'd with Rob on what if anything should remain. isFlashSupported/flashVersion are the only things staying.